WalletWasabi/WabiSabi/Models/MultipartyTransaction/ConstructionState.cs

```diff
@@ -72,6 +72,13 @@ public ConstructionState AddOutput(TxOut output)
 			throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.NotEnoughFunds);
 		}
 
+		return AddOutputCore(output);
+	}
+
+	public ConstructionState AddOutputBypassMinAmount(TxOut output) => AddOutputCore(output);
```

Comment:
This is a synonymous. Instead rename the Core as Nocheck or something like that.